### PR TITLE
calcite-action disabled property

### PR DESCRIPTION
**Pull request by driskull**
**Related Issue:** #162

## Summary

Enhancement: disabled property on `calcite-action`

### CODE REVIEW

**Comment by pr3tori4n** [comment]
Should there be a test for this?

**Comment by driskull** [comment]
Added test!

File: src/components/calcite-action/calcite-action.scss [hunk 1]
```diff
@@ -50,6 +50,15 @@
   background-color: var(--calcite-app-background-active);
 }
 
+:host([disabled]) .button,
+:host([disabled]) .button:hover {
+  cursor: default;
+  background-color: gray;
+  fill: white;
+  color: white;
+  /* todo for Alan */
```
Comment (jcfranco) [inline src/components/calcite-action/calcite-action.scss]: Use issue/PR instead of comment?

Comment (driskull) [reply src/components/calcite-action/calcite-action.scss]: https://github.com/Esri/calcite-app-components/issues/170

**Pull request merged**